### ISSUE TITLE
Add jumper to JP1 external power jumper

### ISSUE THREAD

**Issue by nseidle** [sparkfun/Simultaneous_RFID_Tag_Reader#6]
To make it easier to swap back and forth

**Comment by alwaysAn0n** [comment]
This would be very useful.  I can't imagine why it doesn't ship with one.

**Comment by ReneElizondo** [comment]
Yes please, that would be incredible useful, but please don't stop there, you  should also add jumpers to the antenna select so that its easy to switch between the external and internal antenna.
